baseapp/options.go

```diff
@@ -32,7 +32,7 @@ func SetSubstoreTracer(w io.Writer) StoreOption {
 
 // SetTracerFor provides a BaseApp option function that sets the
 // tracer for a substore with given skey inside a multistore.
-func SetTracerFor(skey storetypes.StoreKey, w io.Writer) StoreOption {
+func SetTracerFor(skey string, w io.Writer) StoreOption {
 	return func(cfg *multi.StoreParams, _ uint64) error {
```

Comment:
Why change from `storetypes.StoreKey` to `string`?

Reply:
For a `StoreKey`, it is newly created in `SetupBaseAppFromParams` using `NewKVStoreKey`. Since `SetTracerFor` is called before that, we don't have access to that newly created `StoreKey` which is why we just simplify it to a `string`. If I were to use `StoreKey` of the previous app, even though it would have the same underlying name, the object itself would be different so `string` is a better shared object than `StoreKey`.